### ISSUE TITLE
Layer component gives Invalid prop warning when `type` is set to `sky`

### ISSUE THREAD

**Issue by elliotwutingfeng** [visgl/react-map-gl#1267]
When declaring a Layer component with the new "sky" type introduced in Mapboxgl v2
```
<Layer id="sky" type="sky" paint={mapStyle.skybox} />
```
the following warning is given.
```
Warning: Failed prop type: Invalid prop `type` of value `sky` supplied to `Layer`,
 expected one of ["fill","line","symbol","circle","fill-extrusion","raster","background","heatmap","hillshade"]
```
However, the sky still renders correctly.

**Comment by Pessimistress** [comment]
Thanks for reporting. Would you open a PR to fix this?

**Comment by Pessimistress** [comment]
Published 6.0.2
